### PR TITLE
Interface: fix gitCommitHash in C API and Fortran

### PR DESCRIPTION
**Pull request by aprokop**

### CODE REVIEW

File: packages/Interface/src/DTK_Fortran_wrap.cpp [hunk 1]
```diff
@@ -227,13 +227,15 @@ template <typename T> T SwigValueInit() {
 
 #include <string>
 
+
```
Comment (dalg24) [inline packages/Interface/src/DTK_Fortran_wrap.cpp]: ?

File: packages/Interface/src/DTK_Fortran_wrap.cpp [hunk 2]
```diff
@@ -227,13 +227,15 @@ template <typename T> T SwigValueInit() {
...
   std::string DTK_string_version() {
     return std::string(DTK_version());
   }
   std::string DTK_string_git_commit_hash() {
     return std::string(DTK_gitCommitHash());
   }
 
+
```
Comment (dalg24) [inline packages/Interface/src/DTK_Fortran_wrap.cpp]: ?

File: packages/Interface/src/DTK_Fortran_API.F90 [hunk 3]
```diff
@@ -19,6 +19,11 @@ module datatransferkit
  public :: DTK_git_commit_hash
  public :: DTK_CellTopology, DTK_TRI_3, DTK_TRI_6, DTK_QUAD_4, DTK_QUAD_9, DTK_TET_4, DTK_TET_10, DTK_TET_11, DTK_HEX_8, &
     DTK_HEX_20, DTK_HEX_27, DTK_PYRAMID_5, DTK_PYRAMID_13, DTK_WEDGE_6, DTK_WEDGE_15, DTK_WEDGE_18, DTK_N_TOPO
+ public :: DTK_initialize
```
Comment (dalg24) [inline packages/Interface/src/DTK_Fortran_API.F90]: Are these changes just blocks moved around?  Why do they occur at all?

**Comment by aprokop** [comment]
@dalg24 Everything was generated automatically, so is unquestionable!

**Pull request merged**